spec/support/page_objects/teacher_interface/new_teacher_form.rb

```diff
@@ -0,0 +1,11 @@
+module PageObjects
```

Comment:
I notice in this PR you've also added a `SignUp` page object which I think is a duplicate of this one, I'd suggest you can remove this page object.